src/candidate-search/query-list/bib.js

```diff
@@ -83,21 +83,47 @@ export function bibTitle(record) {
   }
 }
 
-// Aleph supports only two queries with or -operator (This is not true,)
 // eslint-disable-next-line max-statements
 export function bibStandardIdentifiers(record) {
 
   const debug = createDebugLogger('@natlibfi/melinda-record-matching:candidate-search:query:bibStandardIdentifiers');
   debug(`Creating queries for standard identifiers`);
 
   const fields = record.get(/^(?<def>020|022|024)$/u);
-  const identifiers = fields.map(toIdentifiers);
-  const pairs = identifiers.reduce(toPairs, []);
+  const identifiers = [].concat(...fields.map(toIdentifiers));
+  const uniqueIdentifiers = [...new Set(identifiers)];
 
-  return pairs.map(([a, b]) => b ? `dc.identifier=${a} or dc.identifier=${b}` : `dc.identifier=${a}`);
+  debug(`Identifiers (${identifiers.length}): ${JSON.stringify(identifiers)}`);
+  debug(`Standard identifier fields: ${JSON.stringify(fields)}`);
+  debug(`Unique identifiers (${uniqueIdentifiers.length}): ${JSON.stringify(uniqueIdentifiers)}`);
```

Comment:
Debugs after length check and debug note when no identifiers are found?

Reply:
Added debug note for case of no identifiers, moved verbose debug data to ':data' namespace.